app/lib/aether/problems.coffee

```diff
@@ -334,7 +334,6 @@ class HintCreator
     @context = context ? {}
 
   getHint: (code, {message, range, error, aether}) ->
-    console.log error
```

Comment:
I think this log tends to be pretty helpful; perhaps the logging should be cleaned up, but would like to preserve it.